src/shared/components/kakaomap/searchCenterMap.tsx

```
@@ -1,10 +1,5 @@
 // src/shared/components/kakaomap/searchCenterMap.tsx
-import {
-  Map,
-  Circle,
-  CustomOverlayMap,
-  useKakaoLoader,
-} from 'react-kakao-maps-sdk';
+import { Map, MapMarker, useKakaoLoader } from 'react-kakao-maps-sdk';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)